selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -74,7 +74,7 @@ public Workspaces(
     String BULK_CHECKBOX = "//md-checkbox[@aria-label='Workspace list']";
     String SEARCH_WORKSPACE_FIELD = "//input[@ng-placeholder='Search']";
     String NO_WORKSPACE_FOUND = "//span[text()='No workspaces found.']";
-    String WORKSPACE_ITEM_NAME = "//div[@id='ws-name-%s']";
+    String WORKSPACE_ITEM_NAME = "//div[contains(@id, '/%s')]";
```

Comment:
This xpath looks way too common - it covers any id started with "/".

Reply:
In this case it has only single element, because this is a part of workspace full name: 
![locator-ws-full-name-dashboard](https://user-images.githubusercontent.com/5408906/59853172-01adbd00-9379-11e9-814a-3f92edac4274.png)

The `"//div[@id='ws-name-%s']`" does not work properly (after the last changes) when to click on workspace that to go to the _Workspaces Details_

Reply:
There is `id="ws-full-name-che/java"` on screenshot. 
Can we use more precise locator:  `"//div[@id='ws-full-name-che/%s']"`?

Reply:
Yes, we can..Only one thing.. in this case we can use this locator only to workspaces of the name _che_  organization.

Reply:
I see. There is class name `class="workspace-name-clip"` can be used as well to locate web element.

Reply:
I propose the compromise solution - to do xpath more precize:
`//div[@class='workspace-name-clip' and contains(@id, '/%s')]`
In this case we have minimal fix up, which works.

If to use full name locator we have a quite big refactoring related to test classes and more part of them are already excluded from test suite (and more it needs to add additional code to get prefix of full name of the workspace.):

```
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/DeleteRunningWorkspaceTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/DeleteStoppingWorkspaceTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/RenameWorkspaceTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceOwnerTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/details/WorkspaceDetailsMachineActionsTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/details/WorkspaceDetailsMachinesRamTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/details/WorkspaceDetailsProjectsTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/details/WorkspaceDetailsSingleMachineTest.java
        modified:   selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/ProjectStateAfterRenameWorkspaceTest.java

```
And it needs to add additional code to get prefix of full name of the workspace.